app/middlewares/users.js

```diff
@@ -0,0 +1,39 @@
+const { validationResult, check } = require('express-validator'),
+  error = require('../errors'),
+  logger = require('../logger');
+
+const validateRequest = (req, res, next) => {
+  const errors = validationResult(req);
+  if (!errors.isEmpty()) {
+    logger.error(JSON.stringify(errors));
```

Comment:
remember never use `JSON.stringify()`, you can only log `logger.error(errors[0]);`